packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1068,4 +1069,41 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
+    function: (element: Element) => {
+      if (element.element === 'parameter') {
+        const parameterLocation = toValue((element as ObjectElement).get('in'));
+        const isInPathItemElement =
+          isArrayElement(element.parent) &&
+          includesClasses(['path-item-parameters'], element.parent);
+
+        if (!isInPathItemElement || parameterLocation !== 'path') {
+          return true;
+        }
+
+        const pathItemElement = element.parent.parent.parent;
+        const isPathItemPartOfPathTemplating = isStringElement(pathItemElement.meta.get('path'));
+
+        if (!isPathItemPartOfPathTemplating) {
+          return true;
+        }
+
+        const pathTemplate = toValue(pathItemElement.meta.get('path'));
+        const parameterName = toValue((element as ObjectElement).get('name'));
+
+        const parseResult = parse(pathTemplate);
```

Comment:
We're missing check if the `path template` is valid:

```suggestion
        const parseResult = parse(pathTemplate);
        if (!parseResult.result.success) return true;
```